### PR TITLE
[GV-61] Docker Dev Live Updates

### PR DESCRIPTION
**Pull request by Connor-Bernard**
### Jira Ticket

[Jira Ticket](https://dashbd.atlassian.net/browse/GV-61)

### Description

Make development easier by adding live updates.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Changes

- Fix issue with how requests were being proxied
- Add live updates

### Testing

Testing done locally

### Checklist

- [x] My branch name matches the format: `<ticket-id>/<brief-description-of-change>`
- [x] My PR name matches the format: `[<ticket-id>] <brief-description-of-change>`
- [x] I have added doc-comments to all new functions ([JSDoc](https://jsdoc.app/) for JS and [Docstrings](https://peps.python.org/pep-0257/) for Python)
- [x] I have reviewed all of my code
- [x] My code only contains major changes related to my ticket

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
+	@cd website/server && npm install
+	@cd website && npm run build
+
+dev-up:
+	@docker compose -f docker-compose.dev.yml up -dV
 
-npm:
-	clear
-	cd api && npm run server
+dev-down:
```
Comment (Mehul-Gandhi) [inline Makefile:13]: Are we replacing manually running `docker compose down` with `dev down`? And likewise for `dev-up`?

**Review by Mehul-Gandhi — APPROVED** [review]
LGTM, but what exactly do we mean by adding "live updates"?

File: Makefile (hunk 1 above)
Comment (Mehul-Gandhi) [reply Makefile:13]: Do we need to update the README for instructions on starting the app?

Comment (Connor-Bernard) [reply Makefile:13]: Yep we're replacing those, and definitely, but I need to refactor the entire readme, so I'm going to do that in GV-62

**Review by Mehul-Gandhi — DISMISSED** [review]
LGTM

**Review by Mehul-Gandhi — APPROVED** [review]
LGTM

**Pull request merged**